docs/tutorials/flight_controller_telemetry_tutorial.rst

```diff
@@ -316,14 +324,18 @@ To learn how to adjust the MOT_POLE_COUNT to properly show RPM, see the :ref:`fc
 
 .. _fc_telemetry_erpm_to_rpm:
 
-Converting ERPM to RPM Workaround
-##################################
+Converting ERPM to RPM Workarounds
+###################################
 As of v1.14.3, PX4 flight controllers expect the DSHOT telemetry messages to match the `KISS ESC standard <https://www.rcgroups.com/forums/showatt.php?attachmentid=8524039&d=1450424877>`_.
-That means that it expects the module to send ERPM/100, while Vertiq modules directly send RPM as covered in :ref:`ESC Telemetry <manual_telemetry>`.
+That means that it expects the module to send ERPM/100, while Vertiq modules, by default, directly send RPM as covered in :ref:`ESC Telemetry <manual_telemetry>`.
 
-To convert from ERPM to RPM, the ERPM value should be divided by the pole count of the motor divided by 2. The flight controller is expecting to receive 
-ERPM/100, so when it receives telemetry it tries to calculate RPM using the following formula shown below. This calculation can also be
-seen in the `PX4 code for DSHOT parsing <https://github.com/PX4/PX4-Autopilot/blob/be03b7f098800d51281142cb0319cc37d36cfed2/src/drivers/dshot/DShot.cpp#L228>`_.
+There are two methods to deal with this. The first, and recommended, option is to configure your module to transmit the ERPM/100 value directly through its telemetry responses. This can be done by changing the value of **Timer Based Protocol RPM Reporting Style** 
+in the Control Center's advanced tab:
+
+.. image:: ../_static/tutorial_images/fc_telemetry_tutorial/erpm_config.png
+    :align: center

```

Comment:
Ditto with the firmware version note and thing about telling people to set their poles